sample_plugin/bessctl_conf/sequential_update.bess

```diff
@@ -0,0 +1,20 @@
+import scapy.all as scapy
+
+eth = scapy.Ether(src='02:1e:67:9f:4d:ae', dst='06:16:3e:1b:72:32')
+ip = scapy.IP(src='192.168.1.1', dst='10.0.0.1')
+udp = scapy.UDP(sport=10001, dport=10002)
+payload = 'helloworld'
+pkt_bytes = bytes(eth/ip/udp/payload)
+
+Source() -> Rewrite(templates=[pkt_bytes]) -> rr::RoundRobin(gates=[1, 2])
+
+# src MAC address
+rr:1 \
+    -> Update(fields=[{'offset': 6, 'size': 6, 'value': 0x1234567890ab}]) \
+    -> Sink()
+
+# UDP src/dst ports
+rr:2 \
+    -> SequentialUpdate(fields=[{'offset': 34, 'size': 2, 'min': 20000, 'max': 30000},
+                      {'offset': 36, 'size': 2, 'min': 40000, 'max': 50000}]) \
+    -> Sink()
```

Comment:
Could you add a use of `add` command below? Just to make sure that not only Init() but also runtime commands work correctly.